tests/src/tests/map.spec.ts

```diff
@@ -120,7 +120,7 @@ export = () => {
 		]);
 		const map2 = new Map<string, number>(arr);
 
-		function values<V>(map: Map<unknown, V>): Array<V> {
+		function values<V extends defined>(map: Map<unknown, V>): Array<V> {
```

Comment:
Breaking typechecker change in TS 4.7, generic types must now extend the class generics too. For Map here, that means it needs to extend defined.